src/App/App.js

```diff
@@ -7,49 +7,43 @@ import CheckOutForm from "../CheckOutForm/CheckOutForm";
 import { Route } from "react-router-dom";
 import { ApolloProvider } from '@apollo/react-hooks';
 import ApolloClient from 'apollo-boost';
+import { CurrentCenterContext } from '../CurrentCenterContext';
 const client = new ApolloClient({
   uri: 'https://safe-space-be.herokuapp.com/graphql',
 });
 
-class App extends Component {
-  constructor() {
-    super();
-    this.state = {
-      reliefCenters: [],
-      reliefCenterID: "4",
-      isLoading: false
-    };
-  }
+const App = () => {
+  const [reliefCenter, setreliefCenter] = useState('');
+  const [isCenterSelected, selectCenter] = useState(false)
+  const [isLoading, setIsLoading] = useState(false);
 
-  selectPin = () => {
-    console.log('test')
-  }
-
-  //add componentDidMount to fetch relief center data
-  // once fetched, set isLoading to false
```

Comment:
Since we're currently fetching data from our Map component, maybe we should get rid of these guys.